packages/verifiable-credentials/test/create-credentials.test.ts

```diff
@@ -0,0 +1,92 @@
+// https://nodejs.org/api/assert.html
+const assert = require('assert');
+import { Utils } from '@verida/client-ts';
+import Credentials from '../src/credentials';
+import SharingCredential from '../src/sharing-credential';
+import { config, connect } from './config'
+
+
+describe('Credential tests', function () {
+    describe('Credential Units', function () {
+        this.timeout(100000);
+        let encryptedData;
+
+
+        let appContext;
+        let shareCredential;
+        let credential;
+
+        it('Login in user and create app context', async function () {
+
+            // create instances
+
+            appContext = await connect(config.PRIVATE_KEY_1);
+
+            shareCredential = new SharingCredential(appContext);
+
+            credential = new Credentials(appContext);
+
+
+            assert.equal(appContext.getContextName(), config.VERIDA_CONTEXT_NAME);
+        });
+        it('Verify Credential JWT was created correctly', async function () {
+
+            const item: any = await credential.createCredentialJWT(config.CREDENTAIL_DATA);
+
+            const testJwt = {
+                didJwtVc: item.didJwtVc
+            }
+
+            assert.deepEqual(item, testJwt, 'Expected credential didJWT was generated');
```

Comment:
This tests a credential was created, but needs more tests to ensure it was generated correctly.

You should:

- Decode the credential and compare the data against the original credential data
- Compare the other credential data with the expected values (ie: issuer, timestamp etc.)

Reply:
Actually, you probably need a function that tests everything about a credential is valid, then re-use that across many of the unit tests.